api/database/seeders/BigSeederPoolCandidateUser.php

```diff
@@ -0,0 +1,121 @@
+        $financeCommunityId = Community::select('id')->where('key', 'finance')->sole()->id;
+
+        $pools = Pool::factory()
+            ->count(2)
+            ->withPoolSkills(4, 4)
+            ->withQuestions(2, 2)
+            ->published()
+            ->withAssessments(3)
+            ->create();
+        $poolIds = $pools->pluck('id')->toArray();
+
+        // User - non-government
+        for ($i = 0; $i < $limit; $i++) {
+            User::factory()
+                ->asApplicant()
+                ->withSkillsAndExperiences()
+                ->afterCreating(function (User $user) use ($poolIds) {
+                    $this->applyToPools($user, $poolIds);
+                })
+                ->count(100)
+                ->create();
+        }
+
+        // User - government
+        for ($i = 0; $i < $limit; $i++) {
+            User::factory()
+                ->asApplicant()
+                ->withSkillsAndExperiences()
+                ->asGovEmployee()
+                ->withEmployeeProfile()
+                ->withCommunityInterests([
+                    array_rand(array_flip([$digitalCommunityId, $atipCommunityId, $financeCommunityId])),
+                ])
+                ->afterCreating(function (User $user) use ($poolIds) {
+                    $this->applyToPools($user, $poolIds);
+                })
+                ->count(100)
+                ->create();
+        }
+
+        // AssessmentResult - Pool one
+        $poolOne = $poolIds[0];
+        $assessmentSteps = AssessmentStep::where('pool_id', $poolOne)->get();
+        $essentialPoolSkills = PoolSkill::where('pool_id', $poolOne)->where('type', PoolSkillType::ESSENTIAL->name)->get();
+        $poolOneCandidatesToFillResults = PoolCandidate::where('pool_id', $poolOne)
+            ->inRandomOrder()
+            ->take(100)
+            ->get();
+        foreach ($poolOneCandidatesToFillResults as $poolOneCandidatesToFillResult) {
+            foreach ($assessmentSteps as $assessmentStep) {
+                if ($assessmentStep->type === AssessmentStepType::APPLICATION_SCREENING->name) {
+                    AssessmentResult::factory()
+                        ->withResultType(AssessmentResultType::EDUCATION)
+                        ->create([
+                            'assessment_step_id' => $assessmentStep->id,
+                            'pool_candidate_id' => $poolOneCandidatesToFillResult->id,
+                        ]);
+                }
+
+                foreach ($essentialPoolSkills as $essentialPoolSkill) {
+                    AssessmentResult::factory()
+                        ->withResultType(AssessmentResultType::SKILL)
+                        ->create([
+                            'assessment_step_id' => $assessmentStep->id,
+                            'pool_candidate_id' => $poolOneCandidatesToFillResult->id,
+                            'pool_skill_id' => $essentialPoolSkill->id,
```

Comment:
For some reason there is no successful justification got seeded.
This results in some mismatching assessment results like 
almost all Successful assessment result got paired with  failed justification

Reply:
![image](https://github.com/user-attachments/assets/d64f73f5-9792-4d32-b7b3-4876801f7b32)